app/helpers/application_helper.rb

```diff
@@ -15,6 +15,12 @@ def format_date(date_obj)
     date_obj.strftime('%d/%m/%Y')
   end
 
+  def format_date_string(date_string)
+    return '' if date_string.nil?
```

Comment:
Could `date_string` ever be empty? e.g `''` If so, this method will blow up

Reply:
Doh. Switched to .empty?

Reply:
I guess you may need to check for nil and empty?